src/components/TokenSelect.js

```diff
@@ -50,7 +51,6 @@ const TokenSelect = (props) => {
 
     return (
       <TouchableHighlight
-        style={index === 0 ? styles.firstItemWrapper : null}
```

Comment:
Why did you remove this style? This changes other places as well

Reply:
We don't need to differentiate the first or the last item anymore.